### ISSUE TITLE
Remove a Player

### ISSUE THREAD

**Issue by dschnaars** [Leek727/ChessSchedule#27]
After a Game has been started and players have joined, the Moderator needs to be able to remove a player from the Game. In such cases where a contestant gets sick or may have joined under 2 different names, this would prevent having to give wins to their opponent in future games.

This could be a button next to the players name "Remove from future rounds" or "Drop player". Should probably ask for confirmation before dropping the player.

**Comment by clayrock78** [comment]
Removing players should not be difficult to implement on the back end. 

Here is the schema...

```
event: 'remove_player'
{
   room_uuid : room_uuid,
   host_uuid : uuid of the room's host (for auth),
   player_uuid : the uuid of the player to remove,
}
returns a "remove_player_res" response,
which follows the standard response schema
```

**Comment by clayrock78** [comment]
Closed in favor of encompassing issue #30
